Add Prometheus projects for communitybridge. (#114)
cncf/soc: communitybridge/README.md

- Relevant Issues: https://github.com/prometheus/prometheus/issues/2935 
- Potential Mentors: Gouthamve @gouthamve 

#### Various React UI improvements and filtering label values API with matchers.

> - Description: Now that Prometheus has the brand new experimental React UI it would  needs various bugfixes and improvements. It is still missing few features from the old UI and as an addition to this want to add support for graph multiple expressions in one graph.

Reviewer: Couple of nits:

```suggestion
- Description: Now that Prometheus has a brand new experimental React UI it would need various bug fixes and improvements. It is still missing a few features from the old UI, and as an addition to this we want to add support for showing multiple expressions in one graph.
```

- Recommended Skills: React, Golang

> - Relevant Issues: https://github.com/prometheus/prometheus/issues/6178 , https://github.com/prometheus/prometheus/issues/1237

Reviewer: ```suggestion
- Relevant Issues: https://github.com/prometheus/prometheus/issues/6178, https://github.com/prometheus/prometheus/issues/1237
```

And https://github.com/prometheus/prometheus/issues/39

> - Relevant Issues: https://github.com/prometheus/prometheus/issues/6178, https://github.com/prometheus/prometheus/issues/1237

Reviewer: And https://github.com/prometheus/prometheus/issues/39

> - Relevant Issues: https://github.com/prometheus/prometheus/issues/6178, https://github.com/prometheus/prometheus/issues/1237

Author: ```suggestion
- Relevant Issues: https://github.com/prometheus/prometheus/issues/6178, https://github.com/prometheus/prometheus/issues/1237, https://github.com/prometheus/prometheus/issues/39
```
